### PR TITLE
updating caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @Aditya202375
- [ ] @ShareenaSyed
- [ ] @SyamalaManam
- [ ] @TuongDuyNguyen
- [ ] @anebri
- [ ] @dpalecek
- [ ] @marktraynor
- [ ] @mwi011
- [ ] @naveenguggilapu
- [ ] @ryansoo33
- [ ] @vvaths
- [ ] @vyas000
- [ ] @asmabegum1202T

### PR DESCRIPTION
**Pull request by vvaths**
fixes #2

### CODE REVIEW

File: _slides/02-vvaths.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Awesome Caption!
```
Comment (vyas000) [inline _slides/02-vvaths.md:6]:
```suggestion
Magical Caption!!!
```

File: _slides/02-vvaths.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
```
Comment (vyas000) [inline _slides/02-vvaths.md:7]: great job!

**Review by vyas000 — CHANGES_REQUESTED** [review]
please make changes requested! looks good otherwise

File: _slides/02-vvaths.md (hunk 2 above)
Comment (vvaths) [reply _slides/02-vvaths.md:7]: thanks!

**Review by vyas000 — APPROVED** [review]
ELGTM!

**Pull request merged**